tests/rbd_mirror/test_rbd_journal_mirroring_with_delay.py

```diff
@@ -0,0 +1,203 @@
+                            delay_per_image=True,
+                            rbd=rbd_obj,
+                            pool=pool,
+                            image=image,
+                            operation="set",
+                        )
+                    else:
+                        rc = config_mirroring_delay(
+                            delay=delay, client=sec_client, operation="set"
+                        )
+                    if rc:
+                        log.error("Setting mirror delay failed")
+                        return 1
+
+                    rc = write_data_and_verify_no_mirror_till_delay(
+                        rbd=rbd_obj,
+                        client=client_node,
+                        sec_rbd=sec_obj,
+                        sec_client=sec_client,
+                        pool=pool,
+                        image=image,
+                        mount_path=f"{mount_path}/file_01",
+                        skip_mkfs=True,
+                        image_config=image_config,
+                        delay=int(delay),
+                    )
+
+                    if rc:
+                        log.error(
+                            f"Verification of no mirror until delay failed for image {image_spec}"
+                        )
+                        return 1
+    except Exception as e:
+        log.error(str(e))
+        return 1
+    finally:
+        if delay_at_primary:
+            rc = config_mirroring_delay(
+                delay_per_image=True,
+                rbd=rbd_obj,
+                pool=pool,
+                image=image,
+                operation="remove",
+            )
+        else:
+            rc = config_mirroring_delay(client=sec_client, operation="remove")
+        if rc:
+            log.error("Removing mirror delay failed")
+            return 1
+    return 0
+
+
+def run(**kw):
+    """CEPH-11491 - [Rbd Mirror] - Delayed Replication - Need
+    to test the limit of delay that can be set. Set a reasonable high delay.
+    Pre-requisites :
+    We need atleast one client node with ceph-common, fio and rbd-nbd packages,
+    conf and keyring files in both clusters with snapshot based RBD mirroring
+    enabled between the clusters.
+    kw:
+        clusters:
+            ceph-rbd1:
+            config:
+                rep_pool_config:
+                    num_pools: 1
+                    num_images: 5
+                    size: 10G
+                    mode: pool # compulsory argument if mirroring needs to be setup
+                    journal_delay:
+                        - "2m"
+                    io_size: 200M
+                ec_pool_config:
+                    num_pools: 1
+                    num_images: 5
+                    size: 10G
+                    mode: pool # compulsory argument if mirroring needs to be setup
+                    journal_delay:
+                        - "2m"
+                    io_size: 200M
+            ceph-rbd2:
```

Comment:
this config is redundant same as `rbd1` please remove it.

Reply:
no its for second cluster and its not redundant. I had replied to the same comment in previous PRs as well.